Podverse/EpisodesTableViewController.swift

```diff
@@ -86,6 +85,7 @@ class EpisodesTableViewController: PVViewController, UITableViewDataSource, UITa
     override func viewDidLoad() {
         super.viewDidLoad()
         loadData()
+        self.tableView.reloadData()
```

Comment:
Its a little weird that we are calling reload data in viewDidLoad. Is there somewhere else we could call this instead? If not, I am ok with it it's just a little a non better practice